test/e2e/common/util.go

```diff
@@ -63,6 +65,51 @@ var CommonImageWhiteList = sets.NewString(
 	imageutils.GetE2EImage(imageutils.Net),
 )
 
+type testImagesStruct struct {
+	BusyBoxImage      string
+	GBFrontendImage   string
+	GBRedisSlaveImage string
+	KittenImage       string
+	LivenessImage     string
+	MounttestImage    string
+	NautilusImage     string
+	NginxImage        string
+	NginxNewImage     string
+	PauseImage        string
+	RedisImage        string
+}
+
+var testImages testImagesStruct
+
+func init() {
+	testImages = testImagesStruct{
```

Comment:
why the move to make this happen inside `init()`?

Reply:
I made the change as a response to your comment: https://github.com/kubernetes/kubernetes/pull/63600#discussion_r229481800

I'm fine either way.

Reply:
I think I prefer the previous way slightly. I think maybe @spiffxp's comment was that the images are now defined "far away" from the tests, though that's fine with me I think.